.tekton/konflux-e2e-tests-pull-request.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: |
       event == "pull_request"
+      && ( !has(body.pull_request) || !body.pull_request.draft)
```

Comment:
> `!has(body.pull_request)`
What does this do?

Reply:
check if the body have pull_request object

Reply:
Can there be a `pull_request` event that does not have a `pull_request` object in the body?

Reply:
should not. At least I dont know it 👯

Reply:
@tnevrlka I think it's needed [due to retest command](https://github.com/konflux-ci/build-definitions/pull/952#discussion_r1572342112)